yoko-testify/src/test/java/testify/hex/HexBuilderTest.java

```diff
@@ -0,0 +1,151 @@
+package testify.hex;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class HexBuilderTest {
+
+    @Test
+    void testBuildHex() {
+        HexBuilder newHexBuildObject = HexBuilder.buildHex();
+        assertEquals(newHexBuildObject.getClass(), HexBuilder.class);
+    }
+
+    @Test
+    void testOctWriting() {
+        HexBuilder newHexBuildObject = HexBuilder.buildHex();
+        int[] listInt = {12, 13, 100, 128};
```

Comment:
Use 0x0c, 0x0D, 0x64, 0x80 for the int literals.

Reply:
Solved